docs/blog/newsletter-november-2017.rst

```diff
@@ -0,0 +1,116 @@
+.. post:: November 05, 2017
+   :tags: newsletter
+
+
+#########################################
+Write the Docs Newsletter – November 2017
+#########################################
+
+Happy November everybody! Kelly O. here, reporting live from the coffee shop that is about to cut me off because I've had three flat whites in a row. But I'm considering them celebratory beverages, because we have a jam-packed month of documentation goodness to talk about.
+
+In the next few weeks, the orangization team will be:
+
+- Announcing the call for proposals (and looking for volunteers) for next year's Portland conference! (If you're not signed up to the conference mailing list, you can do so in `the sidebar of the WTD homepage <http://www.writethedocs.org/>`_.)
```

Comment:
I got so annoyed with this a while back that I made http://www.writethedocs.org/newsletter/

Reply:
OOh, brilliant! ... is there any way we can update the copy to include mailing lists, though? 😬 

... like so: https://github.com/writethedocs/www/pull/352

Reply:
Merged

Reply:
s/orangization/organization (line 11)

Unless, of course, we're all going orange this fall ...

Also I do not understand why the GH review UI is not getting me add a comment at the correct line, but ...